controller/bindings/connman/connman.mli

```diff
@@ -123,6 +125,10 @@ module Service : sig
 
   val set_manual_proxy : t -> string -> unit Lwt.t
 
+  val set_ipv4 : ?address:string -> ?netmask:string -> ?gateway:string -> t -> method':string -> unit Lwt.t
```

Comment:
The `address`, `netmask` and `gateway` parameters are only necessary when `method` is being set to `manual`. I chose for simplicity here, but we could introduce a union type to improve type safety.

Reply:
While there is only 1 function, this function has to deal with options.

Suggesting to follow the proxy way for consistency:

- `set_dhcp_ipv4`
- `set_manual_ipv4`

Or to use a union type, and to do the same with `set_xxx_proxy` methods.